src/cloud/config_manager/controllers/server.go

```diff
@@ -52,6 +52,7 @@ type Server struct {
 	atClient            atpb.ArtifactTrackerClient
 	deployKeyClient     vzmgrpb.VZDeploymentKeyServiceClient
 	vzFeatureFlagClient VizierFeatureFlagClient
+	vzmgrClient         vzmgrpb.VZMgrServiceClient
```

Comment:
We'll need to instantiate vzmgrClient (otherwise it will be nil here).
To do that, we'll need to update the constructor to take the vzmgrClient: https://github.com/pixie-io/pixie/blob/main/src/cloud/config_manager/controllers/server.go#L58 similar to how we do `deployKeyClient`.
And then we need to update https://github.com/pixie-io/pixie/blob/main/src/cloud/config_manager/config_manager_server.go#L96 to initiate vzMgrClient and pass it to the constructor.